frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -192,6 +228,12 @@ function FileManagerHistory:onShowHist()
         _manager = self,
     }
 
+    if match_table then
+        self.filter_text = match_table.filter_text
```

Comment:
The whole thing was a bit hard to review because of the choice of these variable names.
You choose `self.filter_text` because we already have a `self.filter` (not fond of "filter" when reading it, but ok, might be just my French influence).
Or may be its just "text" that reads usually as something we will display, not that we will search. You use "keywords" elsewhere, may be `filter_keywords` would sound better and consistent (even if keywords sounds like its multiple keywords and not  a single string).
You pass them in a `match_table` (again, no real need to put "table" in its name:)), that you could have named `filter_table` or `filtering_info`.

Reply:
We can polish them all for consistency.

Reply:
As you feel (you're the code owner of all this :) and if it doesn't bug anybody else, fine with me.)